src/core/lib/channel/context.h

```diff
@@ -28,9 +28,7 @@
 /// Call context is represented as an array of \a grpc_call_context_elements.
 /// This enum represents the indexes into the array, where each index
 /// contains a different type of value.
-typedef enum {
-  GRPC_CONTEXT_COUNT
-} grpc_context_index;
+typedef enum { GRPC_CONTEXT_COUNT } grpc_context_index;
```

Comment:
nit: is this change necessary to include in this PR? If not could you dis-include it?

Reply:
It's for sanity test to pass, will rebase.